### ISSUE TITLE
run name and layer names incorrect in Tensorboard

### ISSUE THREAD

**Issue by werner-rammer** [tflearn/tflearn#534]
I use latest TFLearn (0.2.2) and TensorFlow (0.12) on Windows (from GIT), and have issues with visualizing results with Tensorboard. 
I use something like:
` 
model.fit({'input': X }, {'target': Y}, n_epoch=99, batch_size=512,
           validation_set=({'input': testX}, {'target': testY}),
           snapshot_step=None, show_metric=True, run_id='BBv44')
`
TensorFlow does write log files in the correct location (in my case: `E:\tmp\tflearn_logs\BBv44`).

*Problem 1*: Runs do not show up in Tensorboard. When I start Tensorboard with `--logdir=E:\tmp\tflearn_logs` it used to show all runs in sub-directories (e.g. BBv40, BBv41, ...). New runs (BBv44) are not in the list. A workaround is to start Tensorboard explicitly with `--logdir=E:\tmp\tflearn_logs\BBv41`. When I do this, data is visualized (but the run has no name). However, I can no longer compare results of multiple runs.

*Problem 2* : The grouping within Tensorboard seems to be messed up. Instead of a category  such as "Accuracy/Validation" I see now groups such as "Adam" with layer image captions like "Adam/Conv2D_1/W/Regularizer/L2-Loss".  From Tensorflow I got the following messages:
`
INFO:tensorflow:Summary name Loss & var loss/ is illegal; using Loss___var_loss/ instead.
INFO:tensorflow:Summary name Loss & var loss/ (raw) is illegal; using Loss___var_loss/__raw_ instead.
INFO:tensorflow:Summary name Loss/ (raw) is illegal; using Loss/__raw_ instead.
INFO:tensorflow:Summary name Conv2D/W/Regularizer/L2-Loss (raw) is illegal; using Conv2D/W/Regularizer/L2-Loss__raw_ instead.
INFO:tensorflow:Summary name Conv2D_1/W/Regularizer/L2-Loss (raw) is illegal; using Conv2D_1/W/Regularizer/L2-Loss__raw_ instead.
INFO:tensorflow:Summary name Conv2D_2/W/Regularizer/L2-Loss (raw) is illegal; using Conv2D_2/W/Regularizer/L2-Loss__raw_ instead.
`
